### PR TITLE
Issue #40: Update of the Changelog

### PR DESCRIPTION
**Pull request by jkremser**
Describing the changes in 1.0.0

### CODE REVIEW

File: CHANGES.rdoc [hunk 1]
```diff
@@ -3,6 +3,22 @@
 This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
+=== V 1.0.0
+* Inventory client has been rewritten to use the cannonical paths - this is a breaking change, more info in link:api_breaking_changes.rdoc
```
Comment (pilhuhn) [inline CHANGES.rdoc]: Add an extra point for the breaking changes

File: api_breaking_changes.rdoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
+== Breaking changes in the major releases
+
+=== 1.0.0
+
+To use the client use the <code>require 'hawkular/hawkular_client'</code> instead of <code>require 'hawkular_all'</code>
```
Comment (pilhuhn) [inline api_breaking_changes.rdoc:5]: 👍  This should perhaps be listed in the top of the 1.0.0 changes as well.

Comment (abonas) [inline api_breaking_changes.rdoc:5]: +1 to highlight it in the main changes doc, this is not an api change, but a naming change.

**Comment by abonas** [comment]
regardless of this PR, worth to take a look at this gem for some automatic changes summary:
https://github.com/skywinder/Github-Changelog-Generator

**Comment by pilhuhn** [comment]
This is boring hitting the :+1: button a second time on https://github.com/hawkular/hawkular-client-ruby/pull/68#issuecomment-218779458 does not show 2\* 👍

**Pull request merged**